### PR TITLE
User drop down

### PR DESCRIPTION
**Pull request by ashleyzeg**

### CODE REVIEW

File: imports/ui/components/Navbar.jsx [hunk 1]
```diff
@@ -30,7 +32,30 @@ class Navbar extends Component {
             <SearchArea />
             <FlatButton label="Agencies" onClick={this.handleAgenciesClicked} />
             <FlatButton label="People" onClick={this.handlePeopleClicked} />
-            <IconButton onClick={this.handleProfileClicked}><Person style={iconStyles} /></IconButton>
+            <IconMenu
```
Comment (ashleyzeg) [inline imports/ui/components/Navbar.jsx]: not sure if this should be moved out into its own component?

File: imports/ui/views/users/UserDropDown/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react'
+import {Popover, Menu, MenuItem} from 'material-ui'
+
+class UserDropDown extends Component {
+
+	render() {
+		return (
+			<Popover
+					open={this.state.open}
```
Comment (bwencke) [inline imports/ui/views/users/UserDropDown/index.js]: odd spacing

File: imports/ui/components/Navbar.jsx (hunk 1 above)
Comment (bwencke) [reply imports/ui/components/Navbar.jsx]: Probably, it is very long

**Review by bwencke — APPROVED** [review]

File: imports/ui/components/Navbar.jsx [hunk 3]
```diff
@@ -30,7 +27,7 @@ class Navbar extends Component {
             <SearchArea />
             <FlatButton label="Agencies" onClick={this.handleAgenciesClicked} />
             <FlatButton label="People" onClick={this.handlePeopleClicked} />
-            <IconButton onClick={this.handleProfileClicked}><Person style={iconStyles} /></IconButton>
+            <UserProfileMenu/>
```
Comment (mhoc) [inline imports/ui/components/Navbar.jsx]: space after the component name

File: imports/ui/components/UserProfileMenu.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  IconButton,
+  IconMenu,
+  MenuItem,
+  Divider
+} from 'material-ui'
+import Person from 'material-ui/svg-icons/action/account-circle'
+import { browserHistory } from 'react-router'
+
+export default class UserProfileMenu extends Component {
+  constructor(props) {
```
Comment (mhoc) [inline imports/ui/components/UserProfileMenu.jsx]: unnecessary

**Review by mhoc — APPROVED** [review]

File: imports/ui/components/UserProfileMenu.jsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      >
+        <MenuItem >
+          <div style={styles.iconMenuViewProfileContainer}>
+            <div style={styles.iconMenuItemContainer}>
+              <IconButton><Person style={styles.iconStyles}/></IconButton>
+            </div>
+            <div style={styles.iconMenuItemContainer}>
+              <div>Name</div>
+              <div>Email</div>
+              <RaisedButton>View Profile</RaisedButton>
```
Comment (garr741) [inline imports/ui/components/UserProfileMenu.jsx]: This makes the UI look a bit weird.

**Review by garr741 — APPROVED** [review]
looks good to me, but i just wanna note that it interrupts the login modal until we get the actual login state handled.

**Pull request closed without merging**